src/nrncvode/netcvode.cpp

```diff
@@ -6123,60 +6115,52 @@ void NetCvode::deliver_net_events(NrnThread* nt) {  // for default method
     nt->_t = tsav;
 }
 
-implementPtrList(PlayRecList, PlayRecord)
-
 void NetCvode::playrec_add(PlayRecord* pr) {  // called by PlayRecord constructor
                                               // printf("NetCvode::playrec_add %p\n", pr);
     playrec_change_cnt_ = 0;
-    prl_->append(pr);
+    prl_->push_back(pr);
 }
 
 void NetCvode::playrec_remove(PlayRecord* pr) {  // called by PlayRecord destructor
                                                  // printf("NetCvode::playrec_remove %p\n", pr);
     playrec_change_cnt_ = 0;
-    int i, cnt = prl_->count();
-    for (i = 0; i < cnt; ++i) {
-        if (prl_->item(i) == pr) {
-            prl_->remove(i);
+    for (auto it = prl_->begin(); it != prl_->end(); ++it) {
+        if (*it == pr) {
+            prl_->erase(it);
```

Comment:
Potentially non-ideal pattern for `std::vector`.

Reply:
The break make it valid